server/src/main/java/org/candlepin/audit/EventBuilder.java

```diff
@@ -121,13 +115,11 @@ public EventBuilder setEventData(Eventful entity) {
                 Map<String, String> eventData = new HashMap<>();
                 eventData.put("subscriptionId", ((Pool) entity).getSubscriptionId());
 
-                try {
-                    event.setEventData(mapper.writeValueAsString(eventData));
-                }
-                catch (JsonProcessingException e) {
-                    log.error("Error while building JSON for pool.created event.", e);
+                String output = Util.toJson(eventData);
+                if (output == null) {
```

Comment:
This seems odd to me.  Shouldn't `Util.toJson` throw an exception if it is unable to serialize to JSON?

Reply:
`Util.toJson` is actually swallowing the exception, logging an error and returning an empty string (don't know why, it might be useful in the other places using this utility method?), so I tried to leave it as it is, and put the burden of throwing the exception here in `EventBuilder`. I did do a mistake though, `if (output == null)` should be `if (output.equals(""))`.

Reply:
I count 3 uses in actual code
1. org.candlepin.audit.Event's constructor
1. org.candlepin.audit.Event's setPrincipal
1. org.candlepin.audit.auth.Principal.toString

All of them are pretty simple.  Would you mind having Util.toJson just being

```java
    public static String toJson(Object anObject) throws JsonProcessingException {
            return mapper.writeValueAsString(anObject);
    }
```

And then those three callers can catch the exception, log it, and set their variables to "".  Meanwhile, EventBuilder can wrap the JsonProcessingException in an IseException and throw it.

Reply:
Done